lib/cocoapods-core/specification/dsl.rb

```diff
@@ -855,15 +855,19 @@ def dependency(*args)
       #
       #   A path to a prefix header file to inject in the prefix header of the
       #   pod project.
+      #   ':skip' indicates that the default CocoaPods prefix header should not
+      #   be generated.
       #
       #   ---
       #
-      #   This attribute is __not recommended__ as Pods should not pollute the
-      #   prefix header of other libraries or of the user project.
+      #   The file path options is __not recommended__ as Pods should not
+      #   pollute the prefix header of other libraries or of the user project.
+      #
       #
       #   @example
       #
       #     spec.prefix_header_file = 'iphone/include/prefix.pch'
+      #     spec.prefix_header_file = ':skip'
```

Comment:
this should probably use a symbol instead of a string? Since `:skip` could very well be a valid file path

Reply:
or `spec.prefix_header_file = false` ?

Reply:
`false` is good. Made that change. Thanks!